src/app/src/main/kotlin/org/vaccineimpact/api/app/controllers/PasswordController.kt

```diff
@@ -58,4 +57,17 @@ open class PasswordController(
         }
         return okayResponse()
     }
+
+    private fun getSetPasswordToken(username: String): String
```

Comment:
Shouldn't this function be moved from some existing location? But I don't see a corresponding deletion.

Reply:
Yes well spotted! I forgot to delete the old method from `AbstractController`, done now